using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -828,15 +935,17 @@ MobileCore.setPushIdentifier(token);
 {% endtab %}
 
 {% tab title="iOS" %}
-### setPushIdentifier
+### setPushIdentifier<a id="setPushIdentifier-ios"></a>
 
 #### **Objective-C**
 
 ```objectivec
 + (void) setPushIdentifier: (nullable NSData*) deviceToken;
 ```
 
-Example
+o *deviceToken* is a String that contains the device token for push notifications.
```

Comment:
- Move to after the code example section.
- * *deviceToken* is a string that contains the device token for push notifications.